template-coq/gen-src/run_extractable.ml

```diff
@@ -0,0 +1,103 @@
+open Extractable
+open Plugin_core
+
+(* todo(gmm): determine what of these already exist. *)
+let to_constr (t : Ast0.term) : Constr.t =
+  failwith "to_constr"
+
+let of_constr (t : Constr.t) : Ast0.term =
```

Comment:
As @mattam82 pointed out, part of this is in `checker/src/term_quoter.ml`. `checker` already depends on `template-coq`. If we import `checker` in this file (`template-coq/gen-src/run_extractable`), we would run into cyclic dependencies between the directories `template-coq` and `checker`. Would this work? If not, should we move `term_quoter` to the `template-coq` directory?

Reply:
Let's move it if we can make it work.